conjure-java-jaxrs-client/src/main/java/com/palantir/conjure/java/client/jaxrs/DialogueFeignClient.java

```diff
@@ -341,26 +352,32 @@ public Exception decode(String _methodKey, feign.Response response) {
         }
     }
 
-    private final class FeignRequestEndpoint implements Endpoint {
-        private final Request request;
-        private final String endpoint;
-        private final HttpMethod method;
+    private final class FeignEndpoint implements Endpoint {
```

Comment:
I've renamed this since it no longer contains a `Request`.

This has the added benefit of being able to differentiate these endpoint implementations before and after this PR if wanted to add an exemption for `TimingEndpointChannel`.

Reply:
I think it may be reasonable to declare an endpoint tag in dialogue to opt into caching, when the endpoint itself isn't an enum constant -- that way dialogue doesn't depend on this library.